joomla/joomla-cms
# [4.0] Add registration menu to login module

This PR adds an option to login module to set a menu item for registration page. Useful to override default registration link "index.php?option=com_users&view=registration" to a custom page by its menu item.

### Summary of Changes
New option in login module.

### Testing Instructions
Apply the PR.

1. Go to front end: nothing should be changed in registration link in login module.
2. Edit login module and set a menu item to X: go to front-end and registration link should point to the desired menu item X.

### Documentation Changes Required
It's a self-explanatory option, anyway the doc page https://docs.joomla.org/Customising_the_Login_Form_module
may cite the new option.
